modules/devguide/examples/python/cloud.py

```diff
@@ -17,7 +17,7 @@
 cb = cluster.bucket(bucket_name)
 cb_coll = cb.default_collection()
 
-# Create a N1QL Primary Index (but ignore if it exists)
+# Create a SQL++ (formerly N1QL) Primary Index (but ignore if it exists)
```

Comment:
Line 31 change is fine, but I think this one may be better simply as "Create a Query Primary Index"  :thinking: